tests/BasicUserAction.gui.ts

```diff
@@ -0,0 +1,96 @@
+import { test } from '@guardianui/test'
+import RTokens from './RTokens.json'
```

Comment:
This list already exist as dependency, use `import rtokens from '@lc-labs/rtokens'`